Makefile

```diff
@@ -30,8 +30,6 @@ debug: ## Output internal make variables
 
 deps: ## Download dependencies
 	go get -u github.com/golang/dep/cmd/dep
-	go get github.com/jteeuwen/go-bindata/...
-	go get github.com/elazarl/go-bindata-assetfs/...
```

Comment:
These are still deps for `dist`. I'm not sure about removing them as it will make it more challenging to build `dist` locally.

Reply:
reversed back to the original repo

Reply:
install it locally once
no need to put into deps that is executed during build but those deps not used anymore